specification/containerregistry/data-plane/Azure.ContainerRegistry/preview/2019-08-15-preview/containerregistry.json

```diff
@@ -1,3091 +1,3091 @@
+                              "type": "string"
+                          }
+                      }
+                  },
+                  "default": {
+                      "description": "ACR error response describing why the operation failed.",
+                      "schema": {
+                          "$ref": "#/definitions/AcrErrors"
+                      }
+                  }
+              }
           },
-          {
-            "$ref": "#/parameters/ImageReference"
+          "patch": {
+              "security": [
+                  {
+                      "registry_auth": [],
+                      "registry_oauth2": []
+                  }
+              ],
+              "tags": [
+                  "ContainerRegistryBlob"
+              ],
+              "description": "Upload a stream of data without completing the upload.",
+              "operationId": "ContainerRegistryBlob_UploadChunk",
+              "x-ms-examples": {
+                  "Upload Blob": {
+                      "$ref": "./examples/UploadBlob.json"
+                  }
+              },
+              "consumes": [
+                  "application/octet-stream"
+              ],
+              "parameters": [
+                  {
+                      "$ref": "#/parameters/RawData"
+                  },
+                  {
+                      "$ref": "#/parameters/NextLink"
+                  }
+              ],
+              "responses": {
+                  "202": {
+                      "description": "The stream of data has been accepted and the current progress is available in the range header. The updated upload location is available in the Location header.",
+                      "headers": {
+                          "Location": {
+                              "description": "Provided location for blob",
+                              "type": "string"
+                          },
+                          "Range": {
+                              "description": "Range indicating the current progress of the upload.",
+                              "type": "string"
+                          },
+                          "Docker-Upload-UUID": {
+                              "description": "Identifies the docker upload uuid for the current request.",
+                              "type": "string"
+                          }
+                      }
+                  },
+                  "default": {
+                      "description": "ACR error response describing why the operation failed.",
+                      "schema": {
+                          "$ref": "#/definitions/AcrErrors"
+                      }
+                  }
+              }
           },
-          {
-            "$ref": "#/parameters/ManifestBody"
-          }
-        ],
-        "responses": {
-          "201": {
-            "description": "The manifest is updated",
-            "schema": {},
```

Comment:
I'm not sure if this is needed for anything?

Reply:
I dont think so, can remove.